package.json

```diff
@@ -21,6 +21,7 @@
     "@radix-ui/react-tabs": "^0.1.5",
     "@sentry/react": "^6.18.1",
     "@sentry/tracing": "^6.18.1",
+    "@types/lodash.isequal": "^4.5.5",
```

Comment:
minor (non-blocking): Can we just add @types/loadash instead of needing for each function. As types are stripped at compile time we will have no worries about bundle bloat for this.

Reply:
Others below that could be removed

Reply:
Or we can move to using the whole of lodash, as long as we import correctly it gets tree shaken